src/Compilers/CSharp/Test/Emit2/Semantics/CollectionExpressionTests.cs

```diff
@@ -34698,32 +34629,154 @@ .locals init (System.Collections.Generic.List<T> V_0,
                 """);
         }
 
-        [Fact]
-        public void List_SingleSpread_CustomCollection_NotICollectionAndNoStructEnumerator()
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/71217")]
+        public void List_SingleSpread_IEnumerable()
```

Comment:
The diff is a mess, I don't know why is that. I just added a chunk of `SingleSpread` tests above `SingleSpread_CustomCollection` cases